pkg/storaged/storage-controls.jsx

```diff
@@ -256,15 +257,18 @@ export class StorageUsageBar extends React.Component {
     render() {
         var stats = this.props.stats;
         var fraction = stats ? stats[0] / stats[1] : null;
+        var labelText = utils.format_fsys_usage(stats[0], stats[1]);
```

Comment:
This needs to be moved into the `if (stats)` branch to avoid accessing `stats` when it is null.

Reply:
Fixed.